fathom/src/source.rs

```diff
@@ -76,6 +114,21 @@ impl StringInterner {
     }
 }
 
+fn alphabetic_name(index: usize) -> String {
+    let base = index / 26;
+    let letter = index % 26;
```

Comment:
It's weird that Rust doesn't have `divmod` particularly since x86 `div` provides both. I suppose LLVM might work that out though.

Ok yes, looks like it can work it out: https://rust.godbolt.org/z/hj9rb4Txa

Reply:
Lol, I think I actually added divmod to the stdlib _years_ ago. Maybe we removed at some stage before 1.0?

Reply:
Amazing! Sadly no match for `div_mod` in the docs now.

Reply:
It was named `div_rem` at some point too! https://github.com/rust-lang/rust/pull/6147/commits/ee26c7c433dbb8d41a2b65dbc89eb84acfc2d311

Reply:
I think it might live in num-rs now 🤔

Reply:
Well this has been a fun history lesson